README.md

```diff
@@ -1,185 +1,195 @@
+<div align=center>
+<picture align=center>
+    <source media="(prefers-color-scheme: dark)" srcset="https://assets.timescale.com/docs/images/timescale-logo-dark-mode.svg">
+    <source media="(prefers-color-scheme: light)" srcset="https://assets.timescale.com/docs/images/timescale-logo-light-mode.svg">
+    <img alt="Timescale logo" >
+</picture>
+</div>
+
+<div align=center>
 
+<h3>TimescaleDB is an extension for PostgreSQL that enables time-series workloads while increasing ingest, query, storage, and analytics performance</h3>
 
-## TimescaleDB
+[![Docs](https://img.shields.io/badge/Read_the_Timescale_docs-black?style=for-the-badge&logo=readthedocs&logoColor=white)](https://docs.timescale.com/)
+[![SLACK](https://img.shields.io/badge/Ask_the_Timescale_community-black?style=for-the-badge&logo=slack&logoColor=white)](https://timescaledb.slack.com/archives/C4GT3N90X)
+[![Try TimescaleDB for free](https://img.shields.io/badge/Try_Timescale_for_free-black?style=for-the-badge&logo=timescale&logoColor=white)](https://console.cloud.timescale.com/signup)
 
-TimescaleDB is an open-source database designed to make SQL scalable for
-time-series data.  It is engineered up from PostgreSQL and packaged as a
-PostgreSQL extension, providing automatic partitioning across time and space
-(partitioning key), as well as full SQL support.
+</div>
+
+TimescaleDB scales PostgreSQL for time-series data with the help of [hypertables](https://docs.timescale.com/use-timescale/latest/hypertables/about-hypertables/). Hypertables are PostgreSQL tables that automatically partition your data by time and space. You interact with a hypertable in the same way as regular PostgreSQL table. Behind the scenes, the database performs the work of setting up and maintaining the hypertable's partitions.
+
+From the perspective of both use and management, TimescaleDB looks and feels like PostgreSQL, and can be managed and queried as
+such. However, it provides a range of features and optimizations that make managing your time-series data easier and more efficient.
```

Comment:
I think that the messaging is not 100% consistent with how we talk about Timescale(DB) on timescale.com

I would propose to quickly coordinate with marketing as well on the messaging as we are now expanding to more use cases than pure time series and we also start talking about Hypercore, our hybrid row-columnar engine. As an example from the website:

> Time series, events, and analytics
>
> For ingesting and querying vast amounts of live data. Our hybrid row-columnar engine makes queries up to 350x faster, ingests 44% faster and reduces storage by 95% over RDS.

Reply:
Please note that I don't include other use cases that we have on TImescale Cloud (like RAG, search, and Al).. The additional events, and (real-time) analytics use cases are fully served by the TImescaleDB extension (this repo)

Reply:
Hi @atovpeko , as Yannis has mentioned here , mostly content from https://www.timescale.com , is it possible to add that link as well somewhere in the README ? Possibly, with a normal title : `Timescale is Postgres made powerful` . 
Then, we won't need to replicate the info here.

Reply:
Hi @pallavisontakke There is a section called "About Timescale" at the bottom, with the tagline and the link to the website :-)